hack/build-images.sh

```diff
@@ -44,9 +44,10 @@ cp -f "${imagedir}/dockerregistry" images/dockerregistry/bin
 
 # builds an image and tags it two ways - with latest, and with the release tag
 function image {
+  set -x
   echo "--- $1 ---"
   docker build -t $1:latest $2
-  docker tag $1:latest $1:${OS_RELEASE_COMMIT}
+  docker tag -f $1:latest $1:${OS_RELEASE_COMMIT}
```

Comment:
@smarterclayton  While iterating during testing, I got a set of errors this fixed.  I can pull it now.